### ISSUE TITLE
[Windows Terminal>Settings>Startup]: Screen reader announces only “New line” when resizing rowscolumns using mouse.

### ISSUE THREAD

**Issue by Saurabh220304** [microsoft/terminal#19693]
### Windows Terminal version

1.25.3571.0

### Windows build number

Windows 11 Version 26H1 (OS Build 28000.1340)

### Steps to reproduce

**Repro Steps:**
1.Launch the Window terminal app.
2.Navigate to the 'New Open Tab' and select the 'Settings' menu item.
3.Settings page will open.
4.Now navigate to 'Startup' and select 'it'.
5.Startup page will open.
6.Now navigate to 'Launch size' group and expand it.
7.Now turn on the narrator using 'CTRL+Win+Enter' keys.
8.Now navigate to the 'Column edit' and change the value by up and down arrow keys.
9.Observe the issue here.

User Experience:
Redundant announcements add to cognitive load, making it more challenging for screen reader users to determine if multiple actions have taken place. This disrupts the expectation for concise and meaningful feedback.

WCAG Reference Link:
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

Attachements:
https://microsoftapc-my.sharepoint.com/:u:/g/personal/v-heparihar_microsoft_com/IQDgM8klwC9eR7X0x7NhWg5QAXYxb81I-OQsqdpefwvRaO8?e=uLezCd

### Expected Behavior

Screen reader should announce meaningful feedback such as: Announcement should reflect the current row/column and updated value after resizing.

### Actual Behavior

When resizing rows or columns using the mouse, the screen reader does not announce the current row and column size or position. Instead, it only announces “New line”, which provides no meaningful context about the resize action.
This behavior occurs during mouse‑based interactions and fails to give users feedback about the updated grid structure.

**Comment by carlos-zamora** [comment]
Tested this in WinUI 2 Gallery. This is an issue with the standard `NumberBox` control. Filed an issue upstream on XAML: https://github.com/microsoft/microsoft-ui-xaml/issues/10945.

Closing as a /duplicate of https://github.com/microsoft/microsoft-ui-xaml/issues/10945
